docs/documentation/patrol-tags.mdx

```diff
@@ -81,6 +81,7 @@ Patrol supports complex tag expressions using logical operators:
 - `&&` - AND operator (run tests with both tags)
 - `!` - NOT operator (exclude tests with this tag)
 
+<Warning>Tag can't start or be just a number</Warning>
```

Comment:
let's write what actually a tag can be, as it's not the only condition. Maybe let's link: xhttps://pub.dev/packages/test#tagging-tests and cite: "Note that tags must be valid Dart identifiers, although they may also contain hyphens."